gallery/package.json

```diff
@@ -25,6 +25,7 @@
     "@babel/runtime": "^7.12.0",
     "@types/classnames": "^2.2.11",
     "@types/debounce": "^1.2.0",
+    "@types/ramda": "^0.27.34",
```

Comment:
This was missing for some time, and for whatever reason didn't act up until some cleaning I did while on this branch